dash_draggable/_imports_.py

```diff
@@ -1,6 +1,5 @@
 from .dash_draggable import dash_draggable
 
-
 __all__ = [
-    "dash_draggable",
-]
+    "dash_draggable"
```

Comment:
I don't think it's useful to modify this file, since it is automatically generated when you run `npm run build:py`.